lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -187,4 +187,26 @@ export class PublicClientApplication extends ClientApplication implements IPubli
             }
         }
     }
+
+    /**
+     * Sets the account to use as the active account. If no account is passed to the acquireToken APIs, then MSAL will use this active account.
+     * @param account 
+     */
+    setActiveAccount(account: AccountInfo | null): void {
+        if (this.broker) {
+            this.broker.setActiveAccount(account);
+        }
+        super.setActiveAccount(account);
+    }
+
+    /**
+     * Gets the currently active account
+     */
+    getActiveAccount(asBroker?: boolean): AccountInfo | null {
```

Comment:
These changes should be in the `ExperimentalClient`, right?

Reply:
See [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/pull/2807/files#diff-a22a6fa751365f69045c4a587ace7c208315decc7f08f8b5af791bc9b00057a2R217-R236)

Reply:
Could we rebase this PR into that one then? Or remove these changes here?

Reply:
The experimental API stuff was decided after these changes were already made, which is why they are in a subsequent PR. I could potentially rebase but I think that is going to cause more issues, and take much longer. Removing these changes isn't possible because we are using these APIs in this PR.